stanford_migrate.module

```diff
@@ -92,7 +92,7 @@ function stanford_migrate_entity_delete(EntityInterface $entity) {
       strpos($destination['plugin'], 'entity:') !== FALSE ||
       strpos($destination['plugin'], 'entity_reference_revisions:') !== FALSE
     ) {
-      list(, $type) = explode(':', $destination['plugin']);
+      [, $type] = explode(':', $destination['plugin']);
```

Comment:
This syntax hurts my brain.

Reply:
Agreed. Some code comments are in order here.

Reply:
yeah i hate that phpstorm wraps short array declarations and list declarations into the same setting. So it does the conversion without me even noticing it.
![image](https://user-images.githubusercontent.com/7185045/107687027-cf8e5480-6c5a-11eb-8dfc-c4a89e7cc40a.png)